src/vs/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -299,7 +299,7 @@ export abstract class AbstractInstallAction extends ExtensionAction {
 			// {{SQL CARBON EDIT}}
 			// Prompt the user that the current ADS version is not compatible with the extension,
 			// return here as in this scenario it doesn't make sense for the user to download manually.
-			if (error && error.code === INSTALL_ERROR_INCOMPATIBLE) {
+			if (error && error.name === INSTALL_ERROR_INCOMPATIBLE) {
```

Comment:
Why are you changing it from code to name?

Reply:
Error does not have any code property to it. 
I can see that it only has name

Reply:
Huh, interesting.

Anyways, it'd be better to make a new superclass error IncompatibleEngineError or something that extends error and check typeof the error object in that case